From bd719c5fd879e96450d81339ff85f525a1ad8a13 Mon Sep 17 00:00:00 2001
From: Frederic Peters <fpeters@0d.be>
Date: Wed, 31 Aug 2011 01:24:21 +0000
Subject: Bug 657765 - Explicitly link libgnomecanvas against libm

We used to implicitly link to libm through gtk+, but that got dropped.

http://mail.gnome.org/archives/desktop-devel-list/2011-August/msg00236.html
---
diff --git a/libgnomecanvas/Makefile.am b/libgnomecanvas/Makefile.am
index 4032c85..bb6366b 100644
--- a/libgnomecanvas/Makefile.am
+++ b/libgnomecanvas/Makefile.am
@@ -59,7 +59,8 @@ MARSHAL_GENERATED =				\
 	gnome-canvas-marshal.h
 
 libgnomecanvas_la_LIBADD = \
-	$(GNOME_PLATFORM_LIBS)
+	$(GNOME_PLATFORM_LIBS) \
+	$(MATH_LIB)
 
 libgnomecanvas_la_LDFLAGS = $(NO_UNDEFINED)
 
--
cgit v0.9.0.2
